src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index.jelly

```diff
@@ -0,0 +1,28 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
+  <s:sample>
+    <div class="app-component-sample app-component-sample--full-width">
+      <div class="jenkins-app-bar jenkins-!-margin-bottom-0">
+        <div class="jenkins-app-bar__content">
+          <h1>Page title</h1>
+        </div>
+        <div class="jenkins-app-bar__controls">
+          <button class="jenkins-button jenkins-button--primary">
+            <l:icon src="symbol-add" />
+            Primary control
+          </button>
+          <button class="jenkins-button">Secondary</button>
+          <button class="jenkins-button">Secondary</button>
+          <t:help href="https://www.jenkins.io" />
```

Comment:
the spacing looks not good

![image](https://user-images.githubusercontent.com/21194782/160347123-0d7ba0e6-b073-4d4a-a13b-baa27bf5e150.png)

is that just the branch I'm using (https://github.com/jenkinsci/jenkins/pull/6395)

Reply:
Fix for that should be in with the help button component merge 👍

Reply:
I think my branch has the help button component update in it